packages/x-components/tests/e2e/cucumber/extra-params.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Extra-params component
+
+  Background:
+    Given a next queries API
+    And   a suggestions API
+    And   a related tags API
+    And   a recommendations API with a known response
+    And   a query tagging API
+    And   a click tagging API
+    And   an add to cart tagging API
+    And   a results API with a known response
+    And   no special config for layout view
+
+  Scenario Outline: 1. Search request includes extra-params from Snippet Config and no-Snippet Config
```

Comment:
I don't understand the last part of this sentence 😅

Reply:
I can imagine why... hahah
My first approach included an ExtraParam as well but then I dismissed it and forgot to delete that part of the title.